lib/govuk_publishing_components/presenters/machine_readable/dataset_schema.rb

```diff
@@ -0,0 +1,34 @@
+module GovukPublishingComponents
+  module Presenters
+    class DatasetSchema
+      attr_reader :page
+
+      def initialize(page)
+        @page = page
+      end
+
+      def structured_data
+        # http://schema.org/Dataset
+        data = CreativeWorkSchema.new(@page).structured_data
+          .merge(description)
+          .merge(name)
+        data["@type"] = "Dataset"
+        data
```

Comment:
It's a real shame we don't have attachments appear as links in the content items.  If we did, this could also set the `distribution` property and link straight to the file.

Reply:
@barrucadu maybe one to add to the backlog?

Reply:
Could be, but worth talking to Simon first - there may be other things we'd like to do that would be unblocked by adding attachments to content items, which could inform how we approach that.  For example, having publishing-api tell asset-manager to make an asset live when the corresponding edition is published.

And to do it properly and consistently, we'd need to implement this in *every* publishing app that does attachments, not just whitehall.